Image refactor to use img as placeholder (#301)
thumbtack/thumbprint: packages/thumbprint-react/CHANGELOG.md

## Unreleased

### Changed

-   [Patch] Simplify `Image` component by remove placeholder `div` and CSS positing so the component behaves more predictably. Fixes double download bug in Edge 18.
-   [Major] Only include `object-fit` CSS in `Image` when `height` is provided. That CSS was previously being added when the `containerAspectRatio` was present. `containerAspectRatio` now only adds placeholder spacing on the `<img>` which is removed `onload`.

> -   [Major] Remove CSS that enforced aspect ratio in `ServiceCardImage` due to changes in `Image` component that no longer support this use case.

Reviewer: Need to add docs to `ServiceCardImage` that says the image must be `8:5`.

Author: Added

## Unreleased

### Changed

-   [Patch] Simplify `Image` component by remove placeholder `div` and CSS positing so the component behaves more predictably. Fixes double download bug in Edge 18.

> -   [Major] Remove `object-fit` CSS from `Image` that was previously being added when the `containerAspectRatio` was present. This prop now only adds placeholder spacing on the `<img>` which is removed onload.

Reviewer: ```suggestion
-   [Major] Only include `object-fit` CSS in `Image` when `height` is provided. That CSS was previously being added when the `containerAspectRatio` was present. `containerAspectRatio` now only adds placeholder spacing on the `<img>` which is removed `onload`.
```

Changed it to be a bit more clear that object-fit CSS sometimes is still included.

-   [Major] Remove CSS that enforced aspect ratio in `ServiceCardImage` due to changes in `Image` component that no longer support this use case.

> ## 5.0.0 - 2019-06-13

Reviewer: There's some other stuff here that isn't relevant to this PR. Everything below this should be unchanged.

Author: Updated with `master`.
